src/packages/piral-oidc/src/setup.ts

```diff
@@ -46,27 +75,70 @@ export interface OidcClient {
  * @param config The configuration for the client.
  */
 export function setupOidcClient(config: OidcConfig): OidcClient {
-  const { restrict = false } = config;
+  const {
+    clientId,
+    clientSecret,
+    identityProviderUri,
+    redirectUri = `${location.origin}/auth`,
+    postLogoutRedirectUri = location.origin,
+    responseType,
+    scopes,
+    restrict = false,
+  } = config;
+  const userManager = new UserManager({
+    authority: identityProviderUri,
+    redirect_uri: redirectUri,
+    silent_redirect_uri: redirectUri,
+    post_logout_redirect_uri: postLogoutRedirectUri,
+    client_id: clientId,
+    client_secret: clientSecret,
+    response_type: responseType,
+    scope: scopes?.join(' '),
+  });
+
+  if (process.env.NODE_ENV === 'development') {
```

Comment:
Should this be published? It adds useful debug logs, but I feel like this might be undesired by some users (as long as everything works).

Reply:
Let's just keep it for the moment as is. Later we could also add an option to disable this (e.g., `logDebug?: boolean`).